lib/div-alg.gi

```diff
@@ -1900,7 +1900,7 @@ for i in [1..k] do
     x:=Filtered([1..k],j-> e[i,j] mod Gcd(m,hrs[j][2]-1) = 0);
     if Length(x) > 0 then
       as:=[];
-      for j in x do
+      for j in Difference([1..k],[i]) do
```

Comment:
This looks to me like an amendment of a previous commit, rather than an independent change. Instead of making a new commit, you can amend the last one:
```
git add lib/div-alg.gi
git commit --amend
```
and then push with `--force` (or `-f` for short) to overwrite changes in the remote branch.

We will be able to absorb this commit into the previous one during rebase. The same is about the next commit, I think.